### ISSUE TITLE
Register / setup MERN stack cloud

### ISSUE THREAD

**Issue by dhowe** [dhowe/spectre#418]
requires payment info from @billposters

**Comment by dhowe** [comment]
ref: https://medium.com/@Keithweaver_/single-click-deploy-a-mern-application-on-a-server-efbcbdf056ca

**Comment by billposters** [comment]
Can you pay this and i can transfer it? Easier and quicker if you can as you know what all this means :D

**Comment by dhowe** [comment]
Much simpler if you just do it — will be very fast

**Comment by billposters** [comment]
sorry i need to push back on this one, this form is not simple: 
https://keithweaver.ca/jumpstart/mern/en-ca
It's your area of expertise and I am happy to send you the exact amount today to cover the costs. is that ok?

**Comment by dhowe** [comment]
This is an ongoing monthly cost -- all you need to do is register for an account on digital-ocean and enter paypal info. It is very simple, and then I will do the rest.

**Comment by billposters** [comment]
Please be conscious of your tone you use in your messages. Sometimes it isn't possible to respond to you instantly due to other workflows etc. I'll send you the Digital Ocean account details via message now.

**Comment by dhowe** [comment]
Lets discuss offline

**Comment by billposters** [comment]
can you provide the info that needs to be included for the Jumpstart form? When I get it I can submit the form and purchase.

**Comment by billposters** [comment]
has the issue with the repository been fixed? If so I can try purchase again!

**Comment by dhowe** [comment]
No. Apparently this approach won't work for the setup we need. So we either have to get a static IP from SX, or do the process manually, which is probably 1-2 days of work to get it all set up correctly. 

As discussed, the former option doesn't help for subsequent shows, nor does it all for post-experience after the show closes...
